docs/api.rst

```diff
@@ -611,6 +611,8 @@ Getting or modifying settings
               "ssf_graph_multiplier": 2,
               "non_sync_exchanges": [{"location": "binance", "name": "binance1"}],
               "cost_basis_method": "fifo",
+          "oracle_penalty_threshold_count": 5,
```

Comment:
still not aligned
![2024-01-16_11-22](https://github.com/rotki/rotki/assets/1658405/613f3b8c-c201-4263-9066-7b17f67f589e)

Reply:
lol, I applied you patch. I thought it'd be fixed